recipes/cuda-nvtx/meta.yaml

```diff
@@ -0,0 +1,121 @@
+    - test -L $PREFIX/targets/{{ target_name }}/lib/libnvToolsExt.so.1      # [linux]
+    - test -f $PREFIX/targets/{{ target_name }}/lib/libnvToolsExt.so.1.0.0  # [linux]
+
+outputs:
+  - name: cuda-nvtx
+    files:
+      - lib/libnv*.so.*                       # [linux]
+      - targets/{{ target_name }}/lib/*.so.*  # [linux]
+    requirements:
+      build:
+        - {{ compiler("c") }}
+        - {{ compiler("cxx") }}
+        - arm-variant * {{ arm_variant_type }}  # [aarch64]
+      host:
+        - cuda-version {{ cuda_version }}
+      run:
+        - {{ pin_compatible("cuda-version", max_pin="x.x") }}
+      run_constrained:
+        - arm-variant * {{ arm_variant_type }}  # [aarch64]
+    # Tests are defined at the top level, due to package/output name conflicts.
+    about:
+      home: https://developer.nvidia.com/cuda-toolkit
+      license_file: LICENSE
+      license: LicenseRef-NVIDIA-End-User-License-Agreement
+      license_url: https://docs.nvidia.com/cuda/eula/index.html
+      summary: A C-based API for annotating events, code ranges, and resources
+      description: |
+        A C-based API for annotating events, code ranges, and resources in your
+        applications. Applications which integrate NVTX can use the Visual Profiler
+        to capture and visualize these events and ranges.
+      doc_url: https://docs.nvidia.com/nvtx/
+
+  - name: cuda-nvtx-dev
+    build:
+      run_exports:
+        - {{ pin_subpackage("cuda-nvtx", max_pin='x') }}
+    files:
+      - lib/libnv*.so                                    # [linux]
+      - lib/pkgconfig                                    # [linux]
+      - targets/{{ target_name }}/lib/*.so               # [linux]
+      - targets/{{ target_name }}/include/nvToolsExt*.h  # [linux]
+      - targets/{{ target_name }}/include/nvtx3          # [linux]
+      - Library/include                                  # [win]
+    requirements:
+      build:
+        - arm-variant * {{ arm_variant_type }}  # [aarch64]
+      host:
+        - cuda-version {{ cuda_version }}
+      run:
+        - {{ pin_compatible("cuda-version", max_pin="x.x") }}
+        - {{ pin_subpackage("cuda-nvtx", exact=True) }}
+      run_constrained:
+        - arm-variant * {{ arm_variant_type }}  # [aarch64]
+    test:
+      commands:
+        - test -L $PREFIX/lib/libnvToolsExt.so                                      # [linux]
+        - test -f $PREFIX/lib/pkgconfig/nvToolsExt*.pc                              # [linux]
+        - test -f $PREFIX/targets/{{ target_name }}/include/nvToolsExtCuda.h        # [linux]
+        - test -f $PREFIX/targets/{{ target_name }}/include/nvtx3/nvToolsExtCuda.h  # [linux]
+        - test -L $PREFIX/targets/{{ target_name }}/lib/libnvToolsExt.so            # [linux]
+        - if not exist %LIBRARY_INC%\nvtx3\nvToolsExt.h exit 1                      # [win]
+    about:
+      home: https://developer.nvidia.com/cuda-toolkit
```

Comment:
Should we link to https://docs.nvidia.com/nvtx/ instead? The docs may be the best source, not sure if there is an official nvtx page. Maybe the [Github source](https://github.com/NVIDIA/NVTX) is better.

Reply:
I think the cuda-toolkit url makes sense for `home` here. We already link to their docs and this will only be the `nvtx` bundled with the CTK.

Reply:
Fine with me.

Reply:
Apologies: https://nvtx.readthedocs.io/en/latest/

Reply:
@shwina That link is for the Python package -- we're not shipping that here. This package is C++ headers/libraries only, so I think the other proposals (https://developer.nvidia.com/cuda-toolkit or https://docs.nvidia.com/nvtx/ or https://github.com/NVIDIA/NVTX) would be preferred.

Reply:
Let's stick with what we have for now. If we decide this needs to be changed, we can open an issue with the proposed change and more context and then follow up from there